lib/smart_answer/erb_renderer/question_options_helper.rb

```diff
@@ -3,5 +3,14 @@ module ErbRenderer::QuestionOptionsHelper
     def options(new_options = nil)
       @options = new_options || @options || {}
     end
+
+    def show_caption?
+      @show_caption = true if @show_caption.nil?
+      @show_caption
+    end
```

Comment:
Ignore this if you like, but I'd prefer:
```ruby
def show_caption?
  return true if @show_caption.nil?

  @show_caption
end
```

Reply:
So I've refactored this as hide_caption to simplify things as you suggested!